can't change timezone for cron

Hi, 

I've a problem with cron and timezone, my timezone is Europe/Paris, and the container have the UTC timezone, so we have 2 hours difference, for the cron i have to set -2 for trigger at the good hours in france,

There is a way for change the timezone of the container ?

i've already tried the TZ environment, puting volumes with /etc/timezone and /etc/localtime but the time doesn't impacted

Hi @andreas-marschke, CRON works fine with the good TZ with the 9.47 beta, however i've an error during task running :
```
time="2024-02-05T10:30:09+01:00" level=info msg="Task 1208 added to queue"
time="2024-02-05T10:30:12+01:00" level=info msg="Set resource locker with TaskRunner 1208"
time="2024-02-05T10:30:12+01:00" level=info msg="Task 1208 removed from queue"
time="2024-02-05T10:30:15+01:00" level=info msg="Stopped running TaskRunner 1208"
time="2024-02-05T10:30:15+01:00" level=info msg="Release resource locker with TaskRunner 1208"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x95a8af]
```

Ok i have this @andreas-marschke : 
```
time="2024-02-05T14:06:16Z" level=info msg="Task 1207 added to queue"
time="2024-02-05T14:06:17Z" level=info msg="Set resource locker with TaskRunner 1207"
time="2024-02-05T14:06:17Z" level=info msg="Task 1207 removed from queue"
time="2024-02-05T14:06:23Z" level=info msg="Stopped running TaskRunner 1207"
time="2024-02-05T14:06:23Z" level=info msg="Release resource locker with TaskRunner 1207"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x95a8af]

goroutine 133 [running]:
github.com/ansible-semaphore/semaphore/db_lib.AnsiblePlaybook.RunPlaybook({0x1d, {0x3, {0xc002a869e2, 0xb}, 0x5, {0xc00296c360, 0x25}, {0xc002a869f0, 0x4}, 0x5b, ...}, ...}, ...)
        /go/src/github.com/ansible-semaphore/semaphore/db_lib/AnsiblePlaybook.go:58 +0x8f
github.com/ansible-semaphore/semaphore/db_lib.(*AnsibleApp).Run(0x0?, {0xc001438000?, 0xc000be7958?, 0x1?}, 0xb18300?, 0xb1b380?)
        /go/src/github.com/ansible-semaphore/semaphore/db_lib/AnsibleApp.go:63 +0x90
github.com/ansible-semaphore/semaphore/services/tasks.(*LocalJob).Run(0xc002a90000, {0xc002b5b110, 0xb}, 0x0?)
        /go/src/github.com/ansible-semaphore/semaphore/services/tasks/LocalJob.go:328 +0x335
github.com/ansible-semaphore/semaphore/services/tasks.(*TaskRunner).run(0xc00045d700)
        /go/src/github.com/ansible-semaphore/semaphore/services/tasks/TaskRunner.go:188 +0x6a6
created by github.com/ansible-semaphore/semaphore/services/tasks.(*TaskPool).Run
        /go/src/github.com/ansible-semaphore/semaphore/services/tasks/TaskPool.go:172 +0x4ca
```

is that better ?

--- a/db_lib/AppFactory.go
+++ b/db_lib/AppFactory.go
@@ -15,6 +15,7 @@ func CreateApp(template db.Template, repository db.Repository, logger lib.Logger
 			Playbook: &AnsiblePlaybook{
 				TemplateID: template.ID,
 				Repository: repository,
+				Logger:     logger,
 			},
 		}
 	default:

